<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathSolver</class>
 <widget class="QMainWindow" name="MathSolver">
  <property name="windowTitle">
   <string>MathSolver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>MathSolver</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="formulaInput">
      <property name="placeholderText">
       <string>Enter your formula</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="selectionButton">
        <property name="text">
         <string>Select</string>
        </property>
        <property name="icon">
         <iconset theme="applications-accessories"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
        <property name="icon">
         <iconset theme="document-send"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>